apps/extension/src/ui/domains/Staking/hooks/bittensor/useGetBittensorValidator.ts

```diff
@@ -0,0 +1,108 @@
+import { useQueries, useQuery } from "@tanstack/react-query"
+import { TAOSTATS_API_KEY, TAOSTATS_BASE_PATH } from "extension-shared"
+
+const fetchBittensorValidator = async (hotkey: string | undefined): Promise<BittensorValidator> => {
+  try {
+    return await (
+      await fetch(`${TAOSTATS_BASE_PATH}/api/validator/latest/v1?hotkey=${hotkey}`, {
```

Comment:
This one does handle errors internally, but the error state doesn't seem like it's properly handled in the calling logic - see comment on `BondForm.tsx`